shell/src/main/java/org/apache/accumulo/shell/commands/GrepCommand.java

```diff
@@ -93,10 +102,31 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
   }
 
   protected void setUpIterator(final int prio, final String name, final String term,
```

Comment:
This method is called for each grep term.  I don't think it needs to resetup the profile iters for each term.

Reply:
yes, code updated to remove this